visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<p>Check date</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@AnnaMaguza could you please check the new version and see if there is something I am missing?

Reply:
<p>Looks good :) </p>